spec/models/generic_event/journey_lockout_spec.rb

```diff
@@ -74,32 +75,5 @@
         expect(described_class.from_event(event)).to be_valid
       end
     end
-
-    context 'when the event does not have locations' do
```

Comment:
This context was verifying the process of building a v2 event from a v1 event where there was a missing location. 

Removing this is fine because if there were events created without a location in the v1 world we would have exploded as the data was coming in: https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/dc6a4b3c88eac51587d802ad3269cd39224a2402/app/services/journey_events/params_validator.rb#L15-L16.

For V1 event endpoints this does reflect that these validations which now exist in the model are now duplicated in both the event params validation services (https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/dry_up_locations/app/services/journey_events/params_validator.rb and https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/dry_up_locations/app/services/move_events/params_validator.rb).

This is fine since we need to handle this behaviour in both kinds of endpoint. We could actually probably just remove the old way of doing it, though probably in a separate PR.